From: Felix Geyer <debfx@fobos.de>
Date: Sun, 9 Apr 2017 21:23:49 +0200
Subject: Makefile.am: install-data-hook depends on install-exec

Calling $(DESTDIR)$(bindir)/vnstat$(EXEEXT) fails when installation of this
program is not yet finished.

Author: Adrian Bunk <bunk@debian.org>
(cherry picked from commit 3a76e18434fb033d8ac80955626fc8e2ca966cfa)
---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 6bbe22f..e156982 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -103,7 +103,7 @@ release: distcheck
 	rm -fr $(PACKAGE_NAME)-$(PACKAGE_VERSION)
 	ls -l $(PACKAGE_NAME)-$(PACKAGE_VERSION).tar.gz
 
-install-data-hook:
+install-data-hook: install-exec
 	$(MKDIR_P) $(DESTDIR)$(sysconfdir)
 if IS_CROSSCOMPILED
 	if [ ! -f $(DESTDIR)$(sysconfdir)/vnstat.conf ]; \
